### DISCUSSION TITLE
3D viewer picture in Portfolio

### DISCUSSION THREAD

**Discussion by manhdatbn93** [academicpages/academicpages.github.io#2187] [Q&A]
Hi all,

Is there anyway to add a 3D viewer model to the portfolio page? The model file, such as STL or GLB.

If so, please let me know how to do it. I want to add some 3D models to make them more attractive.

Thanks,
Dat

**Comment by rjzupkoii** [comment]
Are the 3D models viewable in the browser? If so you may have more options, however, on my own site (which has some web applications in the portfolio) I just include a screenshot of project and a link to the live site.
